### PR TITLE
Correct link

### PR DESCRIPTION
**Pull request by eugbaranov**

### CODE REVIEW

**Review by enjoy-binbin — APPROVED** [review]
looks good, thanks

File: docs/manual/data-types/_index.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Sorted Sets are one of the more advanced Redis data types, so take some time to
 Redis also supports Bitmaps and HyperLogLogs which are actually data types
 based on the String base type, but having their own semantics.
 
-Please refer to the [data types tutorial](/topics/data-types-tutorial) for information about those types.
+Please refer to the [data types tutorial]/docs/manual/data-types/data-types-tutorial) for information about those types.
```
Comment (zuiderkwast) [inline docs/manual/data-types/_index.md:145]: `(` is missing

```suggestion
Please refer to the [data types tutorial](/docs/manual/data-types/data-types-tutorial) for information about those types.
```

**Review by zuiderkwast — COMMENTED** [review]
It's annoying that all the URLs have changed. We should at least have 301 redirects for the old URLs.

**Review by zuiderkwast — APPROVED** [review]

**Comment by zuiderkwast** [comment]
The CI is broken so I don't want to merge anything right now.

Another idea: We can add "aliases" to make the old links work. Example: https://github.com/redis/redis-doc/pull/1832/files

**Comment by oranagra** [comment]
it looks like we do have redirection, and the link in https://redis.io/docs/manual/data-types/ is working.

**Comment by oranagra** [comment]
@itamarhaber @banker what was the plan? I assume we've set up a redirection so that we don't need to start converting all the files to the new paths.

**Comment by zuiderkwast** [comment]
It's better to have the links point to the right place rather than to a redirection. But I too would like to know what was the plan.

**Pull request merged**